src/BenchmarkDotNet.Diagnostics.Windows/Tracing/NativeMemoryLogParser.cs

```diff
@@ -23,18 +27,32 @@ public class NativeMemoryLogParser
 
         private readonly ILogger logger;
 
-        public NativeMemoryLogParser(string etlFilePath, BenchmarkCase benchmarkCase, ILogger logger)
+        private readonly (string module, Regex function)[] benchmarkedFunctionStacks;
+
+        public NativeMemoryLogParser(string etlFilePath, BenchmarkCase benchmarkCase, ILogger logger,
+            string programName)
         {
             this.etlFilePath = etlFilePath;
             this.benchmarkCase = benchmarkCase;
             this.logger = logger;
+
+            var module = Path.GetFileNameWithoutExtension(benchmarkCase.Descriptor.WorkloadMethod.Module.Name);
+            var function = new Regex($@"{benchmarkCase.Descriptor.WorkloadMethod.DeclaringType?.FullName}\.{benchmarkCase.Descriptor.WorkloadMethod.Name}\(");
```

Comment:
`type.FullName` is most probably going to return some unusable name for generic types ;/

Reply:
I need more time to check it.

Reply:
@WojciechNagorski ping

Reply:
@adamsitnik I'm going to finish this task but currently I have a lot of work. Unfortunately, BDN is just my hobby.

Reply:
@adamsitnik I've removed this Regex at all.